### PR TITLE
Update Ubuntu depexts for conf-neko

### PR DESCRIPTION
**Pull request by aantron**
It seems both `neko` and `neko-dev` are needed.

cc @andyli

### CODE REVIEW

**Comment by andyli** [comment]
Good catch. I always forgot neko-dev doesn't depends on neko..

File: packages/conf-neko/conf-neko.1/opam [hunk 1]
```diff
@@ -9,9 +9,9 @@ build: [["neko" "-version"]]
 depexts: [
   [["alpine"] ["neko-dev"]]
   [["debian"] ["neko-dev"]]
```
Comment (andyli) [inline packages/conf-neko/conf-neko.1/opam]: Would you also add `neko` to debian?

Comment (aantron) [reply packages/conf-neko/conf-neko.1/opam]: That would be my guess, but I don't have a Debian system to verify on.

Comment (andyli) [reply packages/conf-neko/conf-neko.1/opam]: I'm certain since the ubuntu package comes from debian.

Comment (aantron) [reply packages/conf-neko/conf-neko.1/opam]: Ok, I added a second commit that adds the Debian depext.

**Pull request merged**